fieldpath/fieldpath-gen/template.go

```diff
@@ -1121,10 +1121,11 @@ func (this *{{.CName}}SinglePath) UnmarshalFirst(buf []byte) ({{.ReturnType}}, e
 				endOfs[position] = offset + int(length)
 			}
 		} else {
-			offset, err = skip(buf, offset, wireType)
+			newOffset, err := skip(buf, offset, wireType)
```

Comment:
this is a workaround for a false positive (see the commit message). the problem is that assigning to a previously defined `err` is ambiguous because it is shadowed above.